react-toggle-switch/Toggle-Switch/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+export function ToggleSwitch() {
+  const [onOff, setOnOff] = useState(false);
+  const isOn = onOff;
```

Comment:
Ah, it looks like you did this to make it more clear what true/false mean for `onOff`. But that makes me wonder why you didn't just call the state variable `isOn`?